livingatlas/scripts/la-pipelines

```diff
@@ -252,20 +256,22 @@ function dwca-avro () {
     SECONDS=0
     logStepStart "DWCA-AVRO conversion" $dr
 
-    dwca_dir="/data/biocache-load/$dr/$dr.zip"
+    dwca_dir="$DWCA_IMPORT/$dr/$dr.zip"
 
     if [[ ! -f  $dwca_dir ]]
     then
-        dwca_dir="/data/biocache-load/$dr.zip"
+        dwca_dir="$DWCA_IMPORT/$dr.zip"
     fi
 
     if [[ ! -f  $dwca_dir ]]
     then
-        dwca_dir="/data/biocache-load/$dr.zip"
+        dwca_dir="$DWCA_IMPORT/$dr.zip"
         log.error "$dwca_dir does not exists on your filesystem."
         exit 1
     fi
 
+    export TMPDIR=$DWCA_TMP
```

Comment:
I don't find the use of `TMPDIR`. Is this necessary?

Reply:
Thanks @vjrj it is in use here:

https://github.com/gbif/gbif-common/blob/master/src/main/java/org/gbif/utils/file/FileUtils.java#L827

The tmp directory is a issue for large DwCA files as the default for sorting is to use `/tmp` which causes issues for large files.

Reply:
Ah, ok. This is why I did find the use with a `grep -r`.

Can you add a minor comment to take into account in the future?

Reply:
lve added a comment  - thank you